### ISSUE TITLE
Deployment Issue: PersonAccounts: Standard PersonAccount Duplicate Rule: SortOrder must be in sequential order from 1

### ISSUE THREAD

**Issue by mariocmourao** [sfb2csolutionarchitects/b2c-crm-sync#81]
When I run the following command: "npm run crm-sync:sf:build" I catch  this error 
![image](https://user-images.githubusercontent.com/64840282/117377705-c87a8980-aecb-11eb-8163-d9b0ed100d4e.png)

My configuration:
```
######################################################################
## Salesforce Platform Configuration Properties
######################################################################
SF_SCRATCHORGPROFILE= personaccounts
SF_SCRATCHORGALIAS=crmsync
SF_SCRATCHORGSETDEFAULT=true
SF_SCRATCHORGFORCEOVERWRITE=true
SF_SCRATCHORGDURATIONDAYS=7
```

Note : if run the same command to "base" profile I can run successfully.

**Comment by b2csolutionarchitect** [comment]
Hi there @mariocmourao -- can you check your package.json and see what version you're running?  I recommend pulling from master and re-trying the personAccount deployment.  This used to be an issue with earlier builds, but has since been addressed (I've deployed to about 10 personAccount scratchOrgs today as part of testing.

If you're able to replicate this issue -- please grab the meta-data describing your duplicate rule configuration from your scratchOrg prior to deployment.  This could be an issue where your duplicate rules are in a different order.  

> A workaround for this would be to create a scratchOrg without deployment, look at the duplicate and matchRules in the org and note their sequence numbers. You can then update the meta-data definitions in b2c-crm-sync to correctly align the the sortOrder for your specific scratchOrg.

**Comment by b2csolutionarchitect** [comment]
We've [extended the duplicateRules implementation](https://github.com/sfb2csolutionarchitects/b2c-crm-sync/tree/release/0.9.0#create-and-deploy-your-duplicate-rules) to include a separate deployment command that queries the org for existing duplicate rules -- and uses their sequence values to drive updates / overwrites.  This has been included in the [v0.9.0 release-branch](https://github.com/sfb2csolutionarchitects/b2c-crm-sync/tree/release/0.9.0).

> There's still a scenario where you may need to delete an existing duplicateRule to provide an updated version.  This could be a bug or limitation of the platform (something we're working through figuring out).  In these scenarios, you'll need to delete the existing rule and re-run the deployment command.
